Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV PYBIN=${PYTHONPATH}/bin \
     PYLIB=${PYTHONPATH}/lib
 
 ENV PATH=${PYBIN}:${PATH} \
-    LD_LIBRARY_PATH=/usr/local/cuda/lib64/stubs:/opt/dali/${DALI_BUILD_DIR}:${PYLIB}:${LD_LIBRARY_PATH}
+    LD_LIBRARY_PATH=/usr/local/cuda/lib64/:/usr/local/cuda/lib64/stubs:/opt/dali/${DALI_BUILD_DIR}:${PYLIB}:${LD_LIBRARY_PATH}
```

Comment:
Why it is needed?

Reply:
it is not needed after all because I am not using this Dockerfile now, but I found that `/usr/local/cuda/lib64` libraries were not found because that dir was not in the path. Should I remove it?

Reply:
For compilation `/usr/local/cuda/lib64/stubs` works, for running and anything that uses libs `/usr/local/cuda/lib64/` is needed indeed. The question is if we want to run anything in this image?

Reply:
I'll remove it